tools/event-subscriber/pkg/util/subscriber.go

```diff
@@ -0,0 +1,112 @@
+package util
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"sync"
+)
+
+// Subscribers Servers API paths
+const (
+	// v1 endpoints
+	SubServer1EventsPath  = "/v1/events"
```

Comment:
(nit) I can't recall that the difference between v1, v2 and v3 was. Can we have a clearer naming? Is it event-mesh or event-bus e.g.

Reply:
again: no code was touched. only copied. Lets fix this later